src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -633,28 +633,38 @@ def calculate_sediment_deposition(
                     if e_prime_i == e_prime_nodata:
                         continue
 
+                    # This condition reflects property A in the user's guide.
                     if downslope_sdr_weighted_sum < sdr_i:
                         # i think this happens because of our low resolution
                         # flow direction, it's okay to zero out.
                         downslope_sdr_weighted_sum = sdr_i
 
                     # these correspond to the full equations for
-                    # dr_i, r_i, and f_i given in the docstring
-                    dr_i = (downslope_sdr_weighted_sum - sdr_i) / (1 - sdr_i)
-                    r_i = dr_i * (e_prime_i + f_j_weighted_sum)
-                    f_i = (1 - dr_i) * (e_prime_i + f_j_weighted_sum)
-
-                    # On large flow paths, it's possible for r_i and f_i to
-                    # have very small negative values that are numerically
+                    # dr_i, t_i, and f_i given in the docstring
+                    if sdr_i == 1:
+                        # This reflects property B in the user's guide and is
+                        # an edge case to avoid division-by-zero.
+                        dr_i = 1
+                    else:
+                        dr_i = (downslope_sdr_weighted_sum - sdr_i) / (1 - sdr_i)
+
+                    # Lisa's modified equations
+                    t_i = dr_i * f_j_weighted_sum  # deposition
```

Comment:
What about adding "trapped sediment" to the comment as well?

`# deposition | trapped sediment`

Reply:
Sure!  Added in a4a0af346